docs/source/technical_tutorials/visualization/stac_ipyleaflet.ipynb

```diff
@@ -0,0 +1,208 @@
+{
```

Comment:
<p>Link to https://stac-browser.maap-project.org</p><p>Do not list specific collections "GEDI, BIOMASS, and NISAR" 2 of those don't exist yet and GEDI isn't in the MAAP catalog.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>